cmake/findDependencies.cmake

```diff
@@ -47,7 +47,9 @@ endif()
 
 if (NOT USE_BUNDLED_TINYXML2)
     find_package(tinyxml2 QUIET)
-    if (NOT tinyxml2_FOUND)
+    if (TARGET tinyxml2::tinyxml2)
+        set(tinyxml2_LIBRARIES "tinyxml2::tinyxml2")
+    else()
```

Comment:
If I understand it correctly this needs to be `elseif (NOT tinyxml2_FOUND)` in case the package does not export a target but was still found.

Reply:
No, the `else` is fine because if `tinyxml2_LIBRARIES` is set (by `find_package` in this case), then `find_library(tinyxml2_LIBRARIES ...)` does nothing.